src/tests/ftest/pool/create_query.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2021-2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from pool_test_base import PoolTestBase
+
+
+class PoolCreateQueryTests(PoolTestBase):
+    # pylint: disable=too-many-ancestors
+    """Pool create tests.
+
+    All of the tests verify pool create performance with 7 servers and 1 client.
+    Each server should be configured with full compliment of NVDIMMs and SSDs.
+
+    :avocado: recursive
+    """
+
+    def test_create_and_query(self):
+        """JIRA ID: DAOS-10339
+
+        Test Description:
+            Create a pool and check that the size of the response is close enough to the requested
+            size.  Verify that the size of pool is equal to the size returned in the response of the
+            pool create.
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,medium,ib2
+        :avocado: tags=pool
+        :avocado: tags=pool_create_tests,create_and_query
+        """
+        # Create pool
+        self.add_pool(namespace="/run/pool/*")
```

Comment:
Minor - Namespace is not required if you're just passing the default
```suggestion
        self.add_pool()
```

Reply:
- [ ] Remove useless param